### PR TITLE
Add languages to interface

### PR DESCRIPTION
**Pull request by dpgraham**
@jlipps let me know when we merge this then I'll make a new release.

### CODE REVIEW

File: assets/locales/hi/translation.json [hunk 1]
```diff
@@ -0,0 +1,319 @@
+{
+  "en": "English",
+  "de": "Deutsch",
+  "ru": "रूसी",
```
Comment (mykola-mokhnach) [inline assets/locales/hi/translation.json]: language names should be translated equal for all locales

**Comment by mykola-mokhnach** [comment]
The new language names should also be added to the main translation resource

**Comment by mykola-mokhnach** [comment]
BTW, the tests seem to be flaky

**Comment by dpgraham** [comment]
@mykola-mokhnach the 'tags' build always fails because it tries publishing the same Windows version twice. I just need to take that out.

I'll ask the translators to do the locale translation.

**Pull request merged**